app/models/host/managed.rb

```diff
@@ -167,6 +167,10 @@ def compute_provides?(attr)
     end
   end
 
+  def fqdn_options
+    facts['fqdn'] || name
```

Comment:
then maybe I should change lookup_value_match to: ` "fqdn=#{fqdn || name}" `
and remove the other changes (adding fqdn_options)?